docs/UserGuide.md

```diff
@@ -132,8 +146,9 @@ Format: `find person KEYWORD [MORE_KEYWORDS]`
 
 Examples:
 * `find person John` returns `john` and `John Doe`
-* `find person john betsy` returns `John Doe`, `Betsy Crowe`<br>
-  ![result for 'find john betsy'](images/findJohnBetsyResult.png)
+* `find person john betsy` returns `John Doe`, `Betsy Crowe`
+
+<br>![result for 'find john betsy'](images/findJohnBetsyResult.png)
```

Comment:
Do you think it will be better to remove the <br> as there is already a empty line between the bullet and image?

Reply:
Will make the changes!